updatecli/scripts/docker-ce-updates.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eu -o pipefail
+
+if ! command -v docker >/dev/null 2>&1
+then
+  echo "ERROR: command line 'docker' required but not found. Exiting."
+  exit 1
+fi
+
+SCRIPT_PATH=$(dirname "$0")
+SCRIPT_PATH=$(cd "$SCRIPT_PATH" && pwd)
+
+docker run --rm --name=updatecli-docker-ce --volume="$SCRIPT_PATH":/scripts/ --entrypoint=bash ubuntu:bionic -c "chmod +x /scripts/fetch-docker-ce-latest-version.sh ; /scripts/fetch-docker-ce-latest-version.sh"
```

Comment:
```suggestion
docker run --rm --name=updatecli-docker-ce --volume="$SCRIPT_PATH":/scripts/ --entrypoint=bash ubuntu:bionic /scripts/fetch-docker-ce-latest-version.sh
```
Nitpicking: no need to add the executable bit to a shell script if you pass this script's location to the command interpreter.